exercises/G-conditionals-2/exercise-4.js

```diff
@@ -8,7 +8,13 @@
   Hint: Google how to check if a string contains a word
 */
 
-function containsCode(sentence) {}
+function containsCode(sentence) {
+  if (sentence.includes("code")) {
```

Comment:
can you simplify this in a similar way to [this](https://github.com/CodeYourFuture/JavaScript-Core-1-Coursework-Week2/pull/362/files#r1039031970)